systest/Makefile

```diff
@@ -7,7 +7,7 @@ clusters ?= 1
 size ?= 10
 level ?= debug
 bootstrap ?= 5m
-node_selector ?=
+node_selector ?= cloud.google.com/gke-nodepool=
```

Comment:
but it wont work on minikube. how does it fail?
i think there should be a way to leave node_selector empty

Reply:
`invalid value "" for flag -node-selector:  must be formatted as key=value`

after i added this line, it started the job

Reply:
it works on minikube after this edit on my computer.

Reply:
> invalid value "" for flag -node-selector: must be formatted as key=value

we should fix this flag https://github.com/spacemeshos/go-spacemesh/blob/develop/systest/testcontext/string_to_string.go#L11-L23 so that it skips empty string properly

> after i added this line, it started the job

as i mentioned it will not work with minikube, since there are not nodes with such selector

Reply:
> it works on minikube after this edit on my computer.

that's weird, it shouldn't. something else is off then

Reply:
> that's weird, it shouldn't. something else is off then

hmmm... i mean it started the pod pod/systest-068ea83e, but doesn't seem to start any nodes...

Reply:
> hmmm... i mean it started the pod pod/systest-068ea83e, but doesn't seem to start any nodes...

yes that makes sense. scheduler can't find any nodes with specified node_selector

Reply:
updated PR. plz take another look. thanks